### PR TITLE
Feature/umbraco13

### PR DESCRIPTION
**Pull request by MHove89**
Preliminary PR of the new version of the Umbraco connector. We have a few deprecation warnings that we might want to address. Lets discuss these.

image

### CODE REVIEW

File: src/Enterspeed.Source.UmbracoCms/App_Plugins/Enterspeed.Dashboard/Dashboard.Sub.Controllers/enterspeed-failedjobs.controller.js [hunk 1]
```diff
@@ -16,7 +16,7 @@
 
     function getConfiguration() {
         vm.loadingConfiguration = true;
-        dashboardResources.getEnterspeedConfiguration()
```
Comment (MHove89) [inline src/Enterspeed.Source.UmbracoCms/App_Plugins/Enterspeed.Dashboard/Dashboard.Sub.Controllers/enterspeed-failedjobs.controller.js:19]: Temp workaround for existing issue being fixed elsewhere

Comment (jesperweber) [reply src/Enterspeed.Source.UmbracoCms/App_Plugins/Enterspeed.Dashboard/Dashboard.Sub.Controllers/enterspeed-failedjobs.controller.js:19]: Check, I guess we should remove this and as you mention fix this in another PR

File: src/Enterspeed.Source.UmbracoCms/App_Plugins/Enterspeed.Dashboard/Dashboard.Sub.Controllers/enterspeed-seed.controller.js [hunk 2]
```diff
@@ -35,7 +35,7 @@
 
     function getConfiguration() {
         vm.loadingConfiguration = true;
-        dashboardResources.getEnterspeedConfiguration()
```
Comment (jesperweber) [inline src/Enterspeed.Source.UmbracoCms/App_Plugins/Enterspeed.Dashboard/Dashboard.Sub.Controllers/enterspeed-seed.controller.js:38]: I guess we should remove this and as you mention fix this in another PR

File: src/Enterspeed.Source.UmbracoCms/Enterspeed.Source.UmbracoCms.csproj [hunk 3]
```diff
@@ -20,7 +20,7 @@
 	</PropertyGroup>
 
 	<ItemGroup>
-		<PackageReference Include="Enterspeed.Source.Sdk" Version="2.0.0" />
+		<PackageReference Include="Enterspeed.Source.Sdk" Version="2.0.1-alpha.3" />
```
Comment (jesperweber) [inline src/Enterspeed.Source.UmbracoCms/Enterspeed.Source.UmbracoCms.csproj:23]: What changes does the 2.0.1-alpha.3 package contain?

**Review by jesperweber — COMMENTED** [review]
Added some comments/questions

**Review by jesperweber — APPROVED** [review]
💪

**Pull request merged**